app/messaging/create-submission-msg.js

```diff
@@ -199,7 +199,7 @@ function getEmailDetails (submission, desirabilityScore, notifyTemplate, agentAp
       projectStarted: submission.projectStarted,
       planningPermission: submission.planningPermission,
       abstractionLicence: submission.abstractionLicence,
-      projectName: submission.businessDetails.projectName,
+      projectName: submission.businessDetails?.projectName,
```

Comment:
I'd expect `submission.businessDetails.projectName` to always be defined as it's a required input in the UI. If it's not there then something has gone wrong upstream in the frontend.

If it was optional we can't send `null` or `undefined` to Notify as it will error, instead we need to send an empty string (e.g. what we do above for `tenancyAgreement`

Reply:
agree done.